kong/plugins/url-redirect/schema.lua

```diff
@@ -0,0 +1,27 @@
+local typedefs = require "kong.db.schema.typedefs"
+local PLUGIN_NAME = "kong-plugin-redirect"
+
+local schema = {
+  name = PLUGIN_NAME,
+  fields = {
+    { consumer = typedefs.no_consumer },
+    { protocols = typedefs.protocols_http },
+
+    { config = {
+        type = "record",
+        fields = {
+          { request_uri = typedefs.header_name {
+              required = true,
+              default = "github.com/resolutionlife/kong-plugin-redirect" } },
```

Comment:
The request URI you get in `ngx.var.request_uri` is actually the path

```suggestion
              default = "/resolutionlife/kong-plugin-redirect" } },
```